README.md

```diff
@@ -344,17 +344,22 @@ The global planner has been so far tested on a Odroid companion computer by the
 
 ## Local Planner
 
-Once the catkin workspace has been built, to run the planner with a Realsense D435 camera launch *local_planner_example.launch* editing the arguments:
+Once the catkin workspace has been built, to run the planner with a Realsense D435 camera you can generate the launch file using the script *generate_launchfile.sh*
 
-1. `tf_*` representing the displacement between the camera and the flight controller
-2. `fcu_url` representing the port connecting the companion computer to the flight controller
-3. `serial_no_camera_front` representing the Realsense serial number
+1. `export CAMERA_CONFIGS="camera_namespace, realsense_serial_n, tf_x, tf_y, tf_z, tf_yaw, tf_pitch, tf_roll"` where `tf_*` represents the displacement between the camera and the flight controller. If more than one camera is present, list the eight parameters of each camera separated by a semicolon.
```

Comment:
Almost. I would state it like: 
"If more than one camera is used, the different camera configurations are separated by semicolons, while the individual values within each configuration are still comma-separated."